sensor_msgs/Image image
sensor_msgs/CameraInfo camera_info
---
apriltag_ros/AprilTagDetectionArray tag_detections